__tests__/components/TransactionTypeSelector.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { getComponent } from '@MOCKS/Components';
+import TransactionTypeSelector from '@/components/TransactionTypeSelector/TransactionTypeSelector.vue';
+import TransactionTypeSelectorTs from '@/components/TransactionTypeSelector/TransactionTypeSelectorTs';
+
+describe('components/TransactionTypeSelector', () => {
+    const getTransactionTypeSelectorWrapper = () => {
+        return getComponent(TransactionTypeSelector, {}, {}, { value: null, disabled: false }, {});
+    };
+
+    test('returns sorted transaction type list', () => {
+        // Arrange:
+        const wrapper = getTransactionTypeSelectorWrapper();
+        const component = wrapper.vm as TransactionTypeSelectorTs;
+
+        // Act:
+        const list = component.transactionTypeList;
+
+        // Assert:
+        expect(list).toEqual([
```

Comment:
isn't ACCOUNT_OPERATION_RESTRICTION missing? 🤔

Reply:
`ACCOUNT_OPERATION_RESTRICTION` was filed out, not sure why 🤔

https://github.com/symbol/desktop-wallet/blob/aa834f28c77bd94d14194e3b25508bf00f41c876/src/components/TransactionTypeSelector/TransactionTypeSelectorTs.ts#L50

Reply:
found out the reason, this `transactionTypeList()` is used for ACCOUNT_OPERATION_RESTRICTION.
so you are able to block/allow all transaction types except itself (ACCOUNT_OPERATION_RESTRICTION)

Reply:
it would be good to add a comment to that affect in the code 👍

Reply:
Sure thing, but Do you think I should rename to `OperationRestrictionTransactionTypeSelector` ?